src/sparseconverter/__init__.py

```diff
@@ -904,175 +907,175 @@ def get(self, item, default):
 # See scripts/benchmark.ipynb to generate this list
 # FIXME improve cost basis based on real use cases.
 _cost = {
-    (CUDA, CUDA): 1.397e-05,
-    (CUDA, CUPY): 0.00290094,
-    (CUDA, CUPY_SCIPY_COO): 0.03676262,
```

Comment:
It's quite hard to review the `_cost` changes - did you notice any obvious outliers or large changes to the previous values?

Reply:
I checked the affected ones, there are no outliers.